.github/workflows/rust.yml

```diff
@@ -0,0 +1,27 @@
+name: macOS + Ubuntu
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  macos-build:
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
+      run: cargo build --examples --verbose
+  
+  ubuntu-build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build
```

Comment:
Pretty sure this step will fail because of the issue mentioned in #40

Reply:
It passes here at least: https://github.com/obelisk/ctap-hid-fido2/pull/1

I thought I might have to manually install libusb (and this PR also require libudev) but I think they are installed by default in the GitHub images.